### PR TITLE
build: Add some classifiers

### PR DESCRIPTION
**Pull request by leestarb**
ordered

## What type of pull request is this?

- [ ] Non-breaking code change
- [ ] Breaking code change
- [x] Documentation change/addition

## Checklist

- [ ] I've formatted my code with [Black](https://black.readthedocs.io/en/stable/)
- [ ] I've ensured my code works on `Python 3.10.x`
- [ ] I've tested my code

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,9 +20,21 @@
...
+        "Intended Audience :: Developers",
         "License :: OSI Approved :: MIT License",
+        "Natural Language :: English",
         "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.10",
+        "Topic :: Software Development :: Libraries",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+        "Topic :: Documentation",
+        "Topic :: Documentation :: Sphinx",
```
Comment (LordOfPolls) [inline setup.py]: We don't use sphinx

File: setup.py [hunk 2]
```diff
@@ -20,9 +20,21 @@
     python_requires=">=3.10",
     install_requires=(Path(__file__).parent / "requirements.txt").read_text().splitlines(),
     classifiers=[
-        "Programming Language :: Python :: 3",
+        "Development Status :: 4 - Beta",
```
Comment (LordOfPolls) [inline setup.py]: We're in alpha, not beta

File: setup.py [hunk 3]
```diff
@@ -20,9 +20,21 @@
...
+        "Development Status :: 5 - Production/Stable",
```
Comment (LordOfPolls) [inline setup.py]: We're in alpha, not stable

**Review by LordOfPolls — CHANGES_REQUESTED** [review]

**Comment by leestarb** [comment]
Now let's merge

**Review by silasary — APPROVED** [review]

**Review by LordOfPolls — APPROVED** [review]

**Comment by i0bs** [comment]
Fun fact: there are around 205 bones in an adult male who is going about their day, but 206 when one is "abnormally excited"

**Pull request merged**